tests/test_tools.py

```diff
@@ -475,6 +475,27 @@ def test_resource_parse(self):
             rsrc_id = resource_id(**test['id_args'])
             self.assertEqual(rsrc_id.lower(), test['resource_id'].lower())
 
+    def test_is_resource_name(self):
+        invalid_names = [
+            '',
+            'knights/ni',
+            'this&that',
+            'i<3you',
+            ''.join('a' for _ in range(261))
+        ]
+
+        for test in invalid_names:
+            self.assertFalse(is_valid_resource_name(test), '"{}" should be invalid'.format(test))
+
+        valid_names = [
+            'abc-123',
+            ' ',  # no one said it had to be a good resource name.
+            ''.join('a' for _ in range(260))
+        ]
+
+        for test in valid_names:
+            self.assertTrue(is_valid_resource_name(test), '"{}" should be valid'.format(test))
```

Comment:
```python
assert is_valid_resource_name(test)
```
You don't need the string, pytest will do it for you
https://docs.pytest.org/en/latest/assert.html

Reply:
Fixed in 1e43d02